server/chat-commands/admin.ts

```diff
@@ -356,14 +367,17 @@ export const commands: ChatCommands = {
 				pageid: `view-bot-${pageid}`,
 			});
 			context.title = `[${user.name}] ${pageid}`;
-			context.send(content);
+			context.send(content, selector);
```

Comment:
```suggestion
			if (selector) {
				context.send(`|selectorhtml|${selector}|${content}`);
			} else {
				context.setHTML(content);
			}
```

Reply:
I had to fix a rebase issue so I manually committed this.